packages/firestore/src/remote/remote_event.ts

```diff
@@ -127,11 +127,12 @@ export class TargetChange {
 
   /**
    * HACK: Views require TargetChanges in order to determine whether the view is
-   * CURRENT, but secondary tabs don't receive remote events. So this method is
-   * used to create a synthesized TargetChanges that can be used to apply a
-   * CURRENT status change to a View, for queries executed in a different tab.
+   * CURRENT, but secondary tabs or new queries mapped to an active target don't
```

Comment:
Nit: Would be easier to parse if you made it "to an already active target".

Reply:
Took your comment below.